components/description/instructions.js

```diff
@@ -0,0 +1,16 @@
+function Instructions(props){
+  const { instructions } = props
+
+  return(
+    <>
+      {instructions.map((instruction, index) => {
```

Comment:
You need a return() after opening the map function brackets {}
Perhaps to test if your component does display data, copy data objects from Mongo Compass, pass them as props to your components